src/main/java/gregtech/common/ClientProxy.java

```diff
@@ -184,6 +184,53 @@ public static void addMaterialFormulaHandler(ItemTooltipEvent event) {
         }
     }
 
+    private static final String[] clearRecipes = new String[]{
+            "quantum_tank",
+            "quantum_chest"
+    };
+
+    @SubscribeEvent
+    public static void addNBTClearingTooltip(ItemTooltipEvent event) {
+        // Quantum Tank/Chest NBT Clearing Recipe Tooltip
+        final EntityPlayer player = event.getEntityPlayer();
+        if (player != null) {
+            InventoryCrafting inv = null;
+            InventoryCraftResult result = null;
+
+            if (player.openContainer instanceof ContainerWorkbench) {
+                inv = ((ContainerWorkbench) player.openContainer).craftMatrix;
+                result = ((ContainerWorkbench) player.openContainer).craftResult;
+            } else if (player.openContainer instanceof ContainerPlayer) {
+                inv = ((ContainerPlayer) player.openContainer).craftMatrix;
+                result = ((ContainerPlayer) player.openContainer).craftResult;
+            }
+
+            if (inv != null) {
+                ItemStack stackResult = result.getStackInSlot(0);
+
+                if (stackResult == event.getItemStack()) {
+                    if (!stackResult.isEmpty() && ItemStack.areItemsEqual(stackResult, event.getItemStack())) {
+                        String unlocalizedName = stackResult.getTranslationKey();
+                        for (String key : clearRecipes) {
```

Comment:
Should this check also include a check the unlocalized name for the mod id, in case there are other mods with the same translation key? The quantum tank/chest have `gregtech` leading off their unlocalized name, so it should be a quick check for that. However, you might want to also include `gtadditions` for Gregicality, depending on if you were planning on adding higher tier tanks/chests.